src/pydvl/valuation/methods/beta_shapley.py

```diff
@@ -0,0 +1,36 @@
+import scipy as sp
+
+from pydvl.valuation.methods.semivalue import SemivalueValuation
+from pydvl.valuation.samplers.base import IndexSampler
+from pydvl.valuation.stopping import StoppingCriterion
+
+__all__ = ["BetaShapleyValuation"]
+
+from pydvl.valuation.utility.base import UtilityBase
+
+
+class BetaShapleyValuation(SemivalueValuation):
+    """Computes Beta-Shapley values."""
+
+    algorithm_name = "Beta-Shapley"
+
+    def __init__(
+        self,
+        utility: UtilityBase,
+        sampler: IndexSampler,
+        is_done: StoppingCriterion,
```

Comment:
This comes out of a discussion with @schroedk: `is_done` sounds like a bool. We should discuss renaming this across all methods to something like `stopping` or `stop_criterion`